packages/react-devtools-core/src/backend.js

```diff
@@ -138,6 +138,7 @@ function setupBackend(wall, resolveRNStyle) {
   var bridge = new Bridge(wall);
   var agent = new Agent(window, {
     rnStyle: !!resolveRNStyle,
+    rnStyleMeasure: !!resolveRNStyle,
```

Comment:
The [`instance.measure`](https://github.com/jhen0409/react-devtools/blob/d4d9ea7cba2c9f47427007d612455d88863d687e/plugins/ReactNativeStyle/setupBackend.js#L67) is from RN [`UIManager.measure`](https://github.com/facebook/react-native/blob/bc1ea548d0017f131c36a30ce06bf4d512cb2f8c/React/Modules/RCTUIManager.m#L1215), it's native method.